### DISCUSSION TITLE
https://plugin-md-enhance.vuejs.press/zh/guide/code/demo/vue.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#860] [Announcements]
# https://plugin-md-enhance.vuejs.press/zh/guide/code/demo/vue.html

格式 注意事项 你只能使用 Vue3。 必须将组件通过 export default 默认导出 我们使用 

https://plugin-md-enhance.vuejs.press/zh/guide/code/demo/vue.html

**Comment by linmingjie1** [comment]
Vue demo throw error: Uncaught SyntaxError: Unexpected token ':'.
